.github/workflows/pr-microservice-test.yml

```diff
@@ -64,6 +64,10 @@ jobs:
           role-to-assume: ${{ secrets.AWS_IAM_ROLE_ARN }}
           aws-region: us-east-1
 
+      - name: Set Memory Map Limit
```

Comment:
This change will break CI test, the code change in GHA workflow won't work until they merged, because we use "pull_request_target".